app/src/main/res/values/strings.xml

```diff
@@ -334,8 +334,8 @@
     <string name="allow_copy_password_title">Clipboard trust</string>
     <string name="allow_copy_password_summary">Allow copying the entry password and protected fields to the clipboard</string>
     <string name="allow_copy_password_warning">Warning: The clipboard is shared by all apps. If sensitive data is copied, other software may recover it.</string>
-    <string name="enable">Enable</string>
-    <string name="disable">Disable</string>
+    <string name="enable">Turn on</string>
+    <string name="disable">Turn off</string>
```

Comment:
I've never seen the Turn On and Turn Off buttons on a popup

Reply:
It is easier, also because "enable" would only offer something if it could be "on", and disabled at the same time. While that could be true, the distinction is almost never important.